app/core/views/users/index.pug

```diff
@@ -1,9 +1,22 @@
-include ../_components/html
-
-+doc()
-  +head()
-  +body()
+doctype html
+html(lang='en')
+  head
+    meta(charset='UTF-8')
+    meta(http-equiv='X-UA-Compatible', content='IE=Edge')
+    meta(content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no', name='viewport')
+    link(href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet')
+    link(href='/css/home.min.css', rel='stylesheet')
+    link(href='/css/form.min.css', rel='stylesheet')
+    script(src='https://code.jquery.com/jquery-3.3.1.min.js')
+    title Express boilerplate
+  body
     .container
       h3= section
       p
         a(href='/') Home
+      form.form-container#form
+        input(type='text', id='name', placeholder='Please enter name', name='name')
+        input(type='email', id='email' placeholder='Please enter name email', name='email')
+        input(type='text', id='city' placeholder='Please enter city', name='city')
```

Comment:
The user model has a reference to cities. Cities, in this case, should have been a drop-down selector with available cities. The connecting reference would be the city id.

From the user model:
```
city: {
  type: mongoose.Schema.Types.ObjectId,
  ref: 'City',
  default: null
}
```